Sources/SwiftParser/Expressions.swift

```diff
@@ -481,20 +481,9 @@ extension Parser {
           arena: self.arena
         )
       )
-    case (._move, let handle)?:
-      let moveKeyword = self.eat(handle)
-      let sub = self.parseSequenceExpressionElement(
-        flavor: flavor,
-        pattern: pattern
-      )
-      return RawExprSyntax(
-        RawConsumeExprSyntax(
-          consumeKeyword: moveKeyword,
-          expression: sub,
-          arena: self.arena
-        )
-      )
+
     case (._borrow, let handle)?:
+      assert(self.experimentalFeatures.contains(.oldOwnershipOperatorSpellings))
```

Comment:
Could you make these `precondition`s? https://github.com/swiftlang/swift-syntax/blob/ee0146217a46f545fa46e019449c1a4763611d1e/Sources/SwiftSyntax/Assert.swift#L13-L22

Reply:
I'm not really sure it worth, although it trivial, The branch is a result of the subject expression, where the `experimentalFeatures` is handled, which is close enough for not checking seriously. Also even if somehow we enter this branch without `oldOwnershipOperatorSpellings` feature in release builds, we still have `atContextualExpressionModifier()` condtion check below so the check is _just to make sure_ the condition check in `ExpressionModifierKeyword.init(lexeme:)` is working.

FWIW, looking at the existing code, `do` expression uses `precodition` but `then` statement uses `assert`.